driver-artemis/src/main/java/io/openmessaging/benchmark/driver/artemis/ArtemisBenchmarkDriver.java

```diff
@@ -49,7 +49,13 @@ public void initialize(File configurationFile, StatsLogger statsLogger) throws I
         log.info("ActiveMQ Artemis driver configuration: {}", writer.writeValueAsString(config));
         try {
             ServerLocator serverLocator = ActiveMQClient.createServerLocator(config.brokerAddress);
-            serverLocator.setConfirmationWindowSize(1000);
+            // confirmation window size is in bytes, set to 1MB
+            serverLocator.setConfirmationWindowSize(1024 * 1024);
+            // use asynchronous sending of messages where SendAcknowledgementHandler reports success/failure
+            serverLocator.setBlockOnDurableSend(false);
```

Comment:
Would this block or reject messages when the client buffer is full?

Reply:
That's a good question. I don't know Artemis so well to answer that. The reason to make this PR was the fact that Artemis Driver didn't seem to use async send or async acknowledgements. When reading the docs/javadocs, I figured that confirmation window size was configured too small and blockOnDurableSend wasn't set. 

It seems that Artemis flow control doc contains some answers: https://activemq.apache.org/components/artemis/documentation/latest/flow-control

It might be the case that the default is unbounded.